src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbsList = document.getElementById('thumbs');
+
+thumbsList.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const target = e.target.closest('a');
+
+  if (target) {
+    largeImg.src = target.href;
```

Comment:
Directly setting `largeImg.src` to `target.href` assumes that `target` is always an anchor element with a valid `href` attribute. Ensure that `target` is indeed an anchor element and `href` is a valid image URL to avoid potential errors.